cmd/kubeadm/test/cmd/BUILD

```diff
@@ -22,7 +22,7 @@ go_test(
         "token_test.go",
         "version_test.go",
     ],
-    args = ["--kubeadm-path=../../kubeadm"],
+    args = ["--kubeadm-path=./../../../../$(location //cmd/kubeadm:kubeadm)"],
```

Comment:
I think you can maybe replace this with `--kubeadm-path=$TEST_SRCDIR/$(location //cmd/kubeadm:kubeadm)`

Reply:
SGTM, just looked up the docs for this. I couldn't find good docs on a reliable runfiles path last night but did find docs that relative should work.

Reply:
https://docs.bazel.build/versions/master/test-encyclopedia.html#initial-conditions was what I referenced.

Reply:
reverted to this, see comments below

Reply:
aha, I figured out the proper way to fix this:
```diff
--- a/cmd/kubeadm/test/cmd/BUILD
+++ b/cmd/kubeadm/test/cmd/BUILD
@@ -22,10 +22,11 @@ go_test(
         "token_test.go",
         "version_test.go",
     ],
-    args = ["--kubeadm-path=../../kubeadm"],
+    args = ["--kubeadm-path=$(location //cmd/kubeadm:kubeadm)"],
     data = ["//cmd/kubeadm"],
     embed = [":go_default_library"],
     importpath = "k8s.io/kubernetes/cmd/kubeadm/test/cmd",
+    rundir = ".",
     tags = [
         "integration",
         "skip",
```

Reply:
(see the description for `rundir` on https://github.com/bazelbuild/rules_go/blob/master/go/core.rst#go_test)

Reply:
Ooh. Nice.

Reply:
confirmed with 0.10 on mac